### PR TITLE
Ajout du lien vers le blog A quoi ça sert

### PR DESCRIPTION
**Pull request by dx44**
Blog domotique généraliste

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 - [Projets DIY](https://projetsdiy.fr) - Plein de guide DIY avec beaucoup d'ESP, d'électronique, des raspberry et de la domotique.
 - [&lt;DEVOTICS/&gt;](https://devotics.fr/) - Blog domotique, objets connectés, nouvelles technologies, développement.
 - [figer](https://www.figer.com/) - Blog de guide et conseil en informatique, réseau et domotique.
+- [A quoi ça sert](https://www.ca-sert-a-quoi.com/tag/domotique/) - Blog avec une section test de matériel domotique.
```
Comment (oncleben31) [inline readme.md:30]:
```suggestion
- [Ça Sert À Quoi ?](https://www.ca-sert-a-quoi.com/tag/domotique/) - Un Blog de mise en situation des nouvelles technologies avec une section domotique.
```
Change with exact name of the blog.
I suggest a new description to be more specific on what wa can find on this blog.

**Review by oncleben31 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (dx44) [reply readme.md:30]: ohhh sorry, I'll stop to pull requests so late in my day :)

Comment (oncleben31) [reply readme.md:30]: Same thing as the other PR. Before resolving the comment you should implement the suggestion or justify what we shouldn't do it. I think you have forgotten to commit the suggestion or to push your modification.

**Review by oncleben31 — APPROVED** [review]
Thank you

**Pull request merged**